package.json

```diff
@@ -23,7 +23,8 @@
     "update-syntax-highlighting": "node ./server/scripts/update-syntax-highlighting.js",
     "update-p5-hinter": "node ./server/scripts/update-p5-hinter.js",
     "heroku-postbuild": "touch .env; npm run build",
-    "storybook:build:css": "node-sass client/styles/main.scss client/styles/storybook.css",
+    "storybook:build:css": "sass client/styles/main.scss client/styles/storybook.css",
+    "migrate-sass": "sass-migrator division 'client/styles/**/*.scss'",
```

Comment:
Thanks so much for working on this!  The migration is a one-time process, right?  So we can probably remove this command now that it is complete?

Reply:
Yes .